src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/GraphqlContextExtensions.kt

```diff
@@ -27,7 +27,7 @@ val GraphQLContext.token: String
     get() = get<String>("token").ifEmpty { throw AccessDeniedException("Token is not set") }
 
 val GraphQLContext.securityContext: Mono<SecurityContext>
-    get() = get("securityContext")
+    get() = get<Mono<SecurityContext>?>("securityContext").cache()
```

Comment:
Verdien blir cachet uendelig?

Reply:
Jeg vil tro at cachen forsvinner så fort referansen forsvinner. Men vi kan godt sette en timeout på cachen også, for sikkerhetsskyld? men hva tenker du den bør stå til da, kanskje 3 eller 5 min?